invisible_cities/core/fit_functions.py

```diff
@@ -27,6 +27,35 @@ def get_errors(cov):
     """
     return np.sqrt(np.diag(cov))
 
+def get_chi2_and_pvalue(ydata, yfit, sigma=None, ndf=1):
+    """
+    Gets reduced chi2 and p-value
+
+    Parameters
+    ----------
+    ydata : np.ndarray
+        Data array
+    ydata : np.ndarray
+        Fit funtion values corresponding to ydata array
+    sigma : np.ndarray
+        Data errors array. If no sigma is given, it takes the poisson case.
+
```

Comment:
ndf is not documented

Reply:
I don't want to be pedantic, but can we replace `len(data)` by `number of data points`?